be/src/olap/reader.h

```diff
@@ -204,6 +209,7 @@ class TabletReader {
     std::vector<bool> _is_lower_keys_included;
     std::vector<bool> _is_upper_keys_included;
     std::vector<ColumnPredicate*> _col_predicates;
+    std::vector<ColumnPredicate*> _all_compound_col_predicates;
```

Comment:
a more intuitive name is better

Reply:
rename to _col_preds_except_leafnode_of_andnode